src/azul/plugins/__init__.py

```diff
@@ -760,6 +760,11 @@ class Checksums(SerializableAttrs):
     s3_etag: str | None = None
     md5: str | None = None
 
+    # REVIEW: This smells. It suggests that there are two kinds of instances,
+    #         complete and incomplete ones, and it is not obvious to me why or
+    #         under what circumstances either of them is used. Please request PL
+    #         to discuss this.
+
```

Comment:
Agreed at PL to override the default implementation of `[to/from]_json` and call the superclass methods instead of adding new methods. There should be only one serialization format. A comment should explain that the omission of optional fields is for backwards compatibility.

Reply:
Also, consider removing `checksums` from `AnvilFile` and just having a field for the `md5` sum.